client/me/purchases/purchases-site/index.tsx

```diff
@@ -76,9 +75,7 @@ export default function PurchasesSite(
 						isDisconnectedSite={ ! site }
 						purchase={ purchase }
 						isJetpack={ isJetpackPlan( purchase ) || isJetpackProduct( purchase ) }
-						isJetpackTemporarySite={
-							purchase.domain === 'siteless.jetpack.com' && isEnabled( 'jetpack/siteless-checkout' )
-						}
+						isJetpackTemporarySite={ purchase.domain === 'siteless.jetpack.com' }
```

Comment:
Not in the scope of this PR, but it looks like we could use the `isJetpackTemporarySitePurchase` function from `me/purchases/utils.js` here. (Flagging in case someone from @Automattic/jetpack-voyager wants to come around and tidy up.)

Reply:
I'll fix this in a follow-up PR. Thanks for the heads-up, Matt!